core/config/project_settings.cpp

```diff
@@ -1114,8 +1114,8 @@ ProjectSettings::ProjectSettings() {
 
 	_add_builtin_input_map();
 
-	// Keep the enum values in sync with the `DisplayServer::ScreenOrientation` enum.
```

Comment:
Should this comment be deleted? Seems unrelated to this PR

Reply:
No, sorry, I deleted that comment accidently.